.gitignore

```diff
@@ -4,4 +4,7 @@ bin/
 !**/src/main/**/bin/
 .idea
 target/
-*.iml
\ No newline at end of file
+*.iml
+.mvn/
+.jython_cache/
+.node_modules/
```

Comment:
we don't need to exclude node_modules from the project because we will not include any frontend logic here